source/precalculus/source/08-TE/05.ptx

```diff
@@ -12,6 +12,464 @@
+        </p>
+      </introduction>
+      <task>
+        <statement>
+          <p>
+            Draw a diagram to represent this situation.
+          </p>
+        </statement>
+        <answer>
+          <p>
+            Students should draw a right triangle where the height is <m>40,000</m> and the base is <m>100,000</m>.
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            The pilot wants to know what angle of fall to take in order to reach the runway. Where is this angle located in your diagram?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            Students may notice that the angle they are looking for is not inside the triangle and the line of sight of the pilot is looking down at the ground.
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Draw a line that represents the line of sight of the pilot. What do you notice about this line and the base of your triangle?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            The pilot's line of sight is parallel to the base of the triangle. 
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            If we know that the pilot's line of sight is parallel to the base of the triangle you created in part (a), then the hypotenuse of the triangle could also be considered a transversal that cuts the two parallel lines.  What angle of the triangle is congruent to the "angle of fall" the pilot wants to take in order to descend the plane <m>100,000</m> feet from the runway?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            The "angle of fall" is congruent to the same angle that is created with the base of the triangle (not the right angle). 
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Now that we know which angle is congruent to the "angle of fall" the pilot needs, which of the <m>6</m> trig functions could we use to find the angle at which she should descend?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            If students label the triangle correctly, they should see that they can now use <m>\tan^{-1}</m> to find the measure of the angle because they have <m>40,000</m> as the opposite side and <m>100,000</m> as the adjacent side.
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Find the measure of the angle the pilot needs to make her initial descent.
+          </p>
+        </statement>
+        <answer>
+          <p>
+            Approximately <m>21.8</m><degree/>.
```

Comment:
Don't planes normally target 3 degree descents?  21.8 is quite steep!

Reply:
Yikes! I didn't think about that! I'll make some number changes.